### ISSUE TITLE
wms spec and autodetect conflict

### ISSUE THREAD

**Issue by jpata** [grid-control/grid-control#8]
In my conf I specify 
~~~
[local]
wms = SLURM
~~~
unfortunately grid-control autodetects a (broken) installation of `qsub` on my machines and prefers that, hence dying with:

~~~
Using batch system: GridEngine (LOCAL)

Exception occured in grid-control [1704]

PluginError: Error while creating instance: Workflow
  PluginError: Error while creating instance: Local
    PluginError: Error while creating instance of type OGE (<class 'grid_control.backends.wms_gridengine.GridEngine'>)
      InstallationError: Could not find file "qconf" in
~~~

Would it be possible to make the configuration option superior to the autodetection?

**Comment by FredStober** [comment]
Hi,
the config option you used was dropped some time ago... You should try
`[global] backend = SLURM`
However I've also re-added the old way to specify the local wms in 62f346a3b756c514898d89aec1f5e81935c9798f .
Cheers, Fred

**Comment by jpata** [comment]
confirmed to be fixed on d2e017d, thanks.
